packages/components/src/components/DaCard.vue

```diff
@@ -216,19 +218,31 @@ export default {
   }
 }
 
-.animate-cards {
-  & .card {
+.card--post {
+  transition: all 0.4s ease-out;
+  transform-origin: center;
+}
 
-    & .card__hover {
-      will-change: transform;
-    }
+.card--post::after {
```

Comment:
According to #78 the `::after` element should have a `16px` left, right and top margins

Reply:
Whoa, this gave me quite some time to work-out, what a wonderful thing to do!